e2e/cypress/integration/team_settings/invite_user_to_closed_team_spec.js

```diff
@@ -63,30 +57,31 @@ describe('Team Settings', () => {
         inviteNewMemberToTeam(newUser.email);
 
         // * Assert that the user has successfully been invited to the team
-        cy.get('.invitation-modal-confirm-sent').should('be.visible').within(() => {
+        cy.get('.invitation-modal-confirm--sent').should('be.visible').within(() => {
             cy.get('.username-or-icon').find('span').eq(0).should('have.text', userDetailsString);
-            cy.get('.InvitationModalConfirmStepRow').find('div').eq(1).should('have.text', inviteSuccessMessage);
+            cy.get('.InviteResultRow').find('div').eq(1).should('have.text', inviteSuccessMessage);
         });
 
         // # Click on the 'Invite More People button'
-        cy.get('.invite-more').click();
+        cy.findByTestId('invite-more').click();
 
         // # Invite a user with an invalid email domain (not sample.mattermost.com)
         inviteNewMemberToTeam(invalidEmail);
 
         // * Assert that the invite failed and the correct error message is shown
-        cy.get('.invitation-modal-confirm-not-sent').should('be.visible').within(() => {
+        cy.get('.invitation-modal-confirm--not-sent').should('be.visible').within(() => {
             cy.get('.username-or-icon').find('span').eq(1).should('have.text', invalidEmail);
-            cy.get('.InvitationModalConfirmStepRow').find('div').eq(1).should('have.text', inviteFailedMessage);
+            cy.get('.InviteResultRow').find('div').eq(1).should('have.text', inviteFailedMessage);
         });
     });
 
     function inviteNewMemberToTeam(email) {
         cy.wait(TIMEOUTS.HALF_SEC);
-        if (isLicensed) {
-            // # Click "Invite members"
-            cy.findByTestId('inviteMembersLink').should('be.visible').click();
-        }
+
+        // if (isLicensed) {
+        //     // # Click "Invite members"
+        //     cy.findByTestId('inviteMembersLink').should('be.visible').click();
+        // }
```

Comment:
Is his intended?

Reply:
I wasn't sure at first so I commented the code, but then forgot to address. Thinking more, it is intentional. This screen does not exist any more, and members is selected by default if it is an available option.